content/en/docs/authentication/_index.md

```diff
@@ -37,53 +37,78 @@ axway auth list [options]
 
 ### login
 
-Logs into the Amplify Platform using Proof Key for Code Exchange (PKCE), client secret key, or signed JSON Web Token (JWT) file.
+Authenticates the {{% variables/axway_cli_prod_name %}} with the {{% variables/platform_prod_name %}} as either a platform account or service account.
 
-You can be logged into multiple accounts with varying client IDs at the same time.
+You can simultaneously log into a platform account (by the web browser) as well as multiple platform and service accounts using distinct client IDs.
```

Comment:
I know you copied my original changes, but "by the web browser" sounds weird. Maybe say "via the web browser" instead?